swmmio/defs/inp_sections.yml

```diff
@@ -63,7 +63,9 @@ inp_file_objects:
   XSECTIONS: [Link, Shape, Geom1, Geom2, Geom3, Geom4, Barrels, XX]
   POLLUTANTS: [Name, MassUnits, RainConcen, GWConcen, I&IConcen, DecayCoeff, SnowOnly,
                CoPollutName, CoPollutFraction, DWFConcen, InitConcen]
+
   INFLOWS: [Node, Constituent, Time Series, Type, Mfactor, Sfactor, Baseline, Pattern]
+  LID_USAGE: [Subcatchment,   'LID_Process',      Number,  Area,       Width,      InitSat,    FromImp,    ToPerv,     RptFile,                  DrainTo,          FromPerv,]
```

Comment:
Please replace this line with the following:
```yml
  LID_USAGE: [Subcatchment, LID_Process, Number, Area, Width, InitSat, FromImp, ToPerv, RptFile, DrainTo, FromPerv,]
```

This edit just removes the extra white space. Thanks!